docs/Actions.md

```diff
@@ -1746,6 +1746,21 @@ It can process unit test results if formatted as junit report as shown in [xctes
 
 ## Misc
 
+### ssh
+
+Lets you execute a series of commands on a remote host, if one of the commands in command-array returns non 0 - it fails.
+
+```ruby
+ssh(
+  host: "dev.januschka.com",
+  username: "root",
+  commands: [
+    "date && true",
+    "date && true"
```

Comment:
Oh because of the return status, right? But isn't date returning the right exit code?